### ISSUE TITLE
NoInvert Background Image - Inversion Fix Editor

### ISSUE THREAD

**Issue by thecoloryes** [alexanderby/darkreader#131]
I've permanently adopted this extension for my personal browsing, and it's 100% saved my eyes. Thank you.

However, I'm having some trouble tweaking Reddit to my liking, even with the Inversion Fix Editor. A lot of subreddits have banner and sidebar images made visible with the background-image property -- they're not actually in `img` tags. Not quite sure what I should add to the "noinvert" list for these in the Inversion Fix Editor, or if if it's possible. Any tips would be great.

**Comment by alexanderby** [comment]
You should pick banner element with element picker in Chrome Dev Tools and choose a CSS selector. Then use "invert": "selector" to invert the banner.
https://github.com/alexanderby/darkreader#using-dev-tools

**Comment by thecoloryes** [comment]
Ahh, thanks for the tip to invert if it's already inverted. I believe I've figured it out now!

**Comment by alexanderby** [comment]
Don't forget to make a Pull Request, so that all users can use your fix.

**Comment by thecoloryes** [comment]
Will do! Want to spend some more tweaking it first, some of my fixes are a little hacky. I want to make sure I've figured something out that works for more than the subreddits I'm subscribed to.

**Comment by thecoloryes** [comment]
Okay, I'm stumped again.

Let's look at https://www.reddit.com/r/knitting

In their sidebar is a picture of Zoidberg knitting. If you inspect the element, the image is actually the `background-url` property on the `side` CSS class. I can easily get the _whole_ sidebar un-inverted by simply adding ".side" to the "invert" list for reddit.com, but that's not what I want.

I suspect the answer lies somewhere in the  `"[style*='background:url'], [style*='background-image:url']",` lines that are part of the `common` declaration. I have tried putting them in both the `invert` and `noinvert` lists for all of reddit, and it doesn't change anything. I tried removing them from the common section and reloading the page, still nothing. If it were in an actual image tag I would just use a selector like ".side > img" but it's not so I can't. I've done enough research to believe that `".side[style*='background-image:url']"` _should_ be the proper selector, except that it's not inline CSS but inherited CSS so it doesn't appear to be picking it up.
